vendor/assets/javascripts/mediaelement/plugins/quality-avalon.js

```diff
@@ -169,12 +169,14 @@
                     : '') +
                     '">') +
                   ((src.title || _quality) + '</label>') +
-                  '</li>';
+                  '</li>';                  
```

Comment:
Could you remove this trailing whitespace please?
```suggestion
                  '</li>';
```